package.json

```diff
@@ -33,7 +39,7 @@
     "test": "NODE_ENV=test jest --detectOpenHandles"
   },
   "dependencies": {
-    "@splitsoftware/splitio": "10.20.0",
+    "@splitsoftware/splitio": "10.22.0",
```

Comment:
Be careful. You are probably using a wrong version of NodeJS/NPM that the one stablished in .nvmrc, because this update should not change the `package-lock.json` file in so many lines.